modules/cluster-logging-release-notes-5.4.0.adoc

```diff
@@ -40,7 +40,7 @@ link:https://access.redhat.com/errata/RHSA-2022:1461[{logging-title-uc} Release
 
 * Before this update, clusters with a large number of namespaces caused Elasticsearch to stop serving requests because the list of namespaces reached the maximum header size limit. With this update, headers only include a list of namespace names, resolving the issue.	(link:https://issues.redhat.com/browse/LOG-1899[LOG-1899])
 
-* Before this update, the OpenShift Logging dashboard showed the number of shards 'x' times bigger than actual value when Elasticsearch has 'x' nodes. This was because it was printing all primary shards for each ES pod and processing sum on it, while the output is always for the whole ES cluster. With this update, the calculation has been corrected.	(link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])	
+* Before this update, the OpenShift Logging dashboard showed the number of shards 'x' times bigger than actual value when Elasticsearch has 'x' nodes. This was because it was printing all primary shards for each ES pod and processing sum on it, while the output is always for the whole ES cluster. With this update, the calculation has been corrected.	(link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])
```

Comment:
```suggestion
* Before this update, the {product-title} Logging dashboard showed the number of shards 'x' times larger than the actual value when Elasticsearch had 'x' nodes. This issue occurred because it was printing all primary shards for each ES pod and calculating a sum on it, although the output was always for the whole ES cluster. With this update, the number of shards is now correctly calculated. (link:https://issues.redhat.com/browse/LOG-2156[LOG-2156])
```
- Use an attribute for the OpenShift product title
- Is OpenShift Logging the name of a UI element on a web page? If so, it should be bold.
- Avoid using `this` on its own without a referent. What does `this` refer to here?
- Unclear what `it` refers to in `it was printing`. The system? the dashboard? the display? Also is `printing` the right word here? Is something being sent to a printer? Or does `print` really mean `output` or `display`?
- Unclear what `it` refers to in `processing sum on it`. 
- Unclear what `while` means in `while the output`. Is is supposed to mean `although` or `when`? Per the ISG: 'Use "while" only to refer to a period of time.'

Reply:
I'm unsure where the space that caused this particular release note to appear to be part of this PR came from, but it is not. This PR is for 5.4.1, this Release Note is for 5.4.0. 

The OpenShift Logging dashboard is a component within the OpenShift Console, the name change does not apply here and using the product name (via attribute or otherwise) would be incorrect.  

'print' in various programming languages is a standard command that directs output. Its usage is correct here.

Reply:
> 
> The OpenShift Logging dashboard is a component within the OpenShift Console, the name change does not apply here and using the product name (via attribute or otherwise) would be incorrect.
> 
 My question wasn't about using an attribute for the logging feature or about the name change for logging, but about whether or not the word `OpenShift` itself should be replaced by the `{product-title}` attribute.